src/gui/entry/EditEntryWidgetBrowser.ui

```diff
@@ -50,6 +50,13 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QCheckBox" name="onlyHttpAuthCheckbox">
+        <property name="text">
+         <string>Use this entry only with HTTP Basic Auth</string>
```

Comment:
We should add a tooltip to this that expands on the setting a little more. Such as:

> Only send this setting to the browser for HTTP Auth dialogs. If enabled, normal login forms will not show this entry for selection.